docs/en/backup-restore.rst

```diff
@@ -2,56 +2,159 @@
+- The User has **changed his mobile device** and need to setup the Wallet Solution in his new mobile device.
+- The User **factory resets** the current phone and needs to set up the Wallet Solution again.
+
+.. note::
+  
+  The Backup and Restore functionality is different from migrating to another Wallet Solution (a.k.a. Data portability). 
+  In the latter case, we are dealing with a scenario in which the User wants to migrate from his current Wallet Solution to a different one due to ceases to support the Wallet Solution as highlighted in Annex 2 of `ARF`_.    
+
+Backup Flow
+-----------
+
+.. _fig_Backup_flow:
+.. figure:: ../../images/Backup_flow.svg
+   :name: Sequence Diagram for Wallet Instance Backup
+   :alt: The figure illustrates the sequence diagram for backup flow, with the steps explained below.
+   :target: https://www.plantuml.com/plantuml/png/VP5HRzGm3CVVyodClMn8j1KmU9YcqxPZe8bDEkaqyG1eIbFlQaYJAd4uAhJlJj96WuvZUMhj_y_-spxrB1s7JejdP9GE3KBBtFlZgd9oLsw9sr07ZqvPmsYuLBQhUYrDOWhFZQQwMXqLwnIwkRwgEkaPNGpThY8XoQ0h-rHVICNMmKsi1TB38dqiXDWCKT_TdjjW6kc6mvtK6dbZTM2oviMaE_3m3d-GmiLp-2KWlltOfV4iZSA8VHe3a2CPpE_1sM6Wt24A6TsTJCezkbggxw4_wsc3Blc8rFaOWhFr9UHW9k_5dEriJHetR9tS9l1w_Cy3mPLLKaFE9fUvnhqG1t3nizUaY47BmGO6sNmBdZiq37VMGMyzfIsHsOfP5oW-jzGqQBuMIvYlHeXnt28c0i4nB4xgvSiIyZGhXv5YajgVLFLo8QBc96aVBs02NvNm0GqwoGWVSO1rwwJ7FsWYKxj9_ReSvzmZVLmT_j_og4mcKyCBezpGCpQGpRydZK_K2pHLU5F-Y-vp_8GKlXY8QTWHjx1sjkkdW_oL6-zQhRGDJRvkzlQm_ld5fePlInZ1ENAAfWcT_Wq0
+   
+   Backup flow.
+
+Below, the description of the steps of :numref:`fig_Backup_flow`:
+
+**Step 1**: The User clicks on the backup credentials option in the Wallet Instance. 
+
+**Steps 2-3**: The Wallet Instance using the Backup APIs randomly selects 10 key phrases from a pre-generated list of words and displays it to the User. 
+The User MUST write down or store the key phrases in a secure place as the backup is encrypted using the generated phrases. 
+
+.. note::
+  
+  As highlighted in the `ARF`_, the reason for encryption is due to the fact that the backup file is considered sensitive. Indeed, even if the attacker knows only the 
+  Issuer identifiers of the Digital Credentials, it enables him to know the different types of Digital Credentials and this is a violation of user privacy.
+
+.. note::
+  
+  To extract the key from the list of selected words a key derivation function MUST be applied. Password-Based-Key-Derivation Function 2 (PBKDF2) is among the mostly used ones based on `RFC 2898`_ and it is recommended by the `NIST 800-132 <https://nvlpubs.nist.gov/nistpubs/Legacy/SP/nistspecialpublication800-132.pdf>`_. There are other relevant techniques that are available and used widely, such as Bcrypt, Scrypt, and Argon2 (used by Hyperledger Indy within Connect.me Wallet). More details on this approach can be found `here <https://cryptobook.nakov.com/mac-and-key-derivation/kdf-deriving-key-from-password>`_.
```

Comment:
Add a note that specifies a suggested value for iterations of PBKDF2 (e.g. 100,000 iterations at least)

Reply:
Added